osu.Game/Screens/Play/Player.cs

```diff
@@ -345,9 +366,42 @@ private bool onFail()
             if (PauseOverlay.State == Visibility.Visible)
                 PauseOverlay.Hide();
 
+            // Disable input to avoid hitting objects while falling
+            GameplayClockContainer.BlockGameplayInput = true;
+
+            return true;
+        }
+
+        private void updateFail()
+        {
+            audio.Track.Frequency.Value -= Game.IsActive.Value ? 0.0015 : 0.00075;
+
+            GameplayClockContainer.UserPlaybackRate.Value -= Game.IsActive.Value ? 0.0015 : 0.00075;
+
+            DrawableRuleset.Playfield.Alpha -= Game.IsActive.Value ? 0.0015f : 0.00075f;
+
+            Random objRand = new Random();
```

Comment:
Use `RNG.`. Also this seems wrong, since the random values are going to change every frame?

Reply:
I'm using RNG now, and i made another calculation that you'll see on the next commit